hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestOzoneConfigurationFields.java

```diff
@@ -49,8 +48,8 @@ public void initializeMemberVariables() {
     configurationClasses =
         new Class[] {OzoneConfigKeys.class, ScmConfigKeys.class,
             OMConfigKeys.class, HddsConfigKeys.class,
+            ReconServerConfigKeys.class,
             ReconConfigKeys.class, ReconServerConfigKeys.class,
```

Comment:
Nit:  `ReconServerConfigKeys` is duplicated.

Reply:
Fixed.  Thanks, somehow slipped in, perhaps one of the rebases?